Form-Controls/style.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: #000;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
+  background-color: #fff;
+}
+header {
+  width: 50%;
+  margin: 100px auto;
+}
+h1 {
+  font-size: 3rem;
+  font-weight: 700;
+  text-align: center;
+}
+
+footer {
+  width: 100%;
+  text-align: center;
+  margin-top: 50px;
+}
+h2 {
+  font-size: 0.8rem;
+  font-weight: 700;
+}
+
+main {
+  width: 100%;
+}
+
+form {
+  width: 450px;
```

Comment:
What happens if the device is smaller than 450px? Would a max-width work in more contexts?